# Perguntas para responder
1.	Calcular a média, mediana, mínimo, máximo e desvio padrão dos valores das vendas e quantidade vendida.
2.	Qual produto que mais vendeu? (Usar a moda para identificar).

### Importando a biblioteca

In [1]:
import pandas as pd

### Importando os dados

In [2]:
df = pd.read_csv('vendas_ficticias.csv')

### Encontrando a média, desvio padrão, mediana, valores mínimos e máximos dos valores das vendas e da quantidade vendida.

In [3]:
df.head()

Data da venda  Valor da venda Vendedor  Quantidade vendida    Produto
0    2024-01-29          212.80    Pedro                   8  Produto D
1    2024-05-19          428.04     João                  16  Produto E
2    2024-02-19          982.79      Ana                  20  Produto A
3    2024-02-15          127.86    Pedro                   1  Produto B
4    2024-01-13          429.88    Maria                   6  Produto E

In [14]:
df.describe().T

count      mean         std    min       25%      50%  \
Valor da venda      100.0  553.2947  281.060737  55.02  335.4675  569.535   
Quantidade vendida  100.0    9.9000    5.880115   1.00    4.0000   10.000   

                         75%     max  
Valor da venda      813.9275  987.42  
Quantidade vendida   15.2500   20.00

##### Em relação as vendas do período, podemos afirmar que:a média de vendas é R$ 553,29 e a média da quantidade vendida é 9,9; a mediana de vendas é R$569,54 e a mediana da quantidade vendida é 10. O menor valor de vendas é R$ 55,02 e a menor quantidade vendida é 1; O maior valor de vendas é R$ 987.42 e a maior quantidade vendida é 20. O desvio padrão de vendas é R$ 281,06 e o desvio padrão da quantidade vendida é 5,88.

### Verificando o produto mais vendido

In [15]:
df['Produto'].mode()

0    Produto D
Name: Produto, dtype: object

##### Em relação as vendas do período, podemos afirmar que o produto D foi o mais vendido. 